### PR TITLE
Add tooltip option for Hide and Copy button and remove the custom Hide and Copy button from Credentials component

### PR DESCRIPTION
**Pull request by niteshseram**
---
name: Add a tooltip option for Hide and Copy button
about: Add a tooltip option for Hide and Copy button and remove the custom Hide and Copy button from the Credentials component
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

There was a custom Hide and Copy button for the Credentials component to just support the tooltip. Instead of doing that, we want the tooltip to be supported in the original Copy and Hide button as well.

Related issue: #994 

### Solution Description

1. Add a tooltip option for Hide and Copy Button.
2. Add TCustomize props to both the Hide and Copy button
3. Remove the custom Hide and Copy button from the Credentials component and replace it with the original Hide and Copy button

### CODE REVIEW

File: packages/core/src/lib/CopyButton/CopyButton.tsx [hunk 1]
```diff
@@ -26,11 +32,33 @@ export const useCopyToClipboard = (): [CopiedValue, CopyFn] => {
...
+    copied ? (
+        <Check
+            title="check icon"
+            titleId="copybutton check icon"
+            fill={color.mint40}
+            fontSize={iconSize}
+        />
+    ) : (
+        <Copy
+            title="copy icon"
```
Comment (niteshseram) [inline packages/core/src/lib/CopyButton/CopyButton.tsx:49]: Hi @AbhinavMV, do you think we need to show the title if we are showing the tooltip here?

**Comment by niteshseram** [comment]
> wow nice job @niteshseram i will check this asap, sorry it is an unusually busy day 😅

Sure @BillyG83 . No problem

**Review by BillyG83 — APPROVED** [review]
all looks good, there was a very minor alignment issue of the buttons with the divider, so please made sure to check that any code you add didnt effect the other stories

**Comment by BillyG83** [comment]
fixed divider
Screenshot 2023-04-24 at 10 47 34

tooltip on hover
Screenshot 2023-04-24 at 10 47 11

**Comment by BillyG83** [comment]
thanks @niteshseram good job. It helps a whole lot of you can add evidence like screenshots are videos to summaries you work PR.

Also i added the changeset file for you, i need to update the docs for this. TY for the contribution!

**Pull request merged**